MetaMask/metamask-extension
# UX: Added deprecation warning for aurora

When we remove Aurora as part of the reconfiguration of the popular custom network list, This PR is to create a warnning banner to inform our users about this change

I have converted the already existing DeprecatedTestNetworks component to the DeprecatedNetworks component so that we can use the same component every time we need to create a warning for new components.

## **Related issues**

Fixes: [#1605](https://github.com/MetaMask/MetaMask-planning/issues/1605)

## **Manual testing steps**

1. Add Aurora mainnet network
2. Switch to Aurora
3. Look at the banner at the bottom informing about the deprecation of Aurora
4. Close the banner it should go away
5. Reload the extension, the banner should show up
6. If close the banner, go to setting page Banner should show up again


## **Screenshots/Recordings**

### **Before**

NA

### **After**

![Screenshot 2023-11-22 at 10 50 07 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/86212bdd-c29f-4e6c-abbd-29bf3f9ff136)

![Screenshot 2023-11-22 at 10 50 21 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/a076f2de-0b6f-442b-a406-2539d9b6e256)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
